test/test_integration_consumer.py

```diff
@@ -0,0 +1,233 @@
+import logging
+import pytest
+import six
+import time
+
+from six.moves import xrange
```

Comment:
per pep8, the above imports should be split into a section for stdlib and a section for 3p imports

Reply:
Will change... And will re-read PEP8!